specs/online_order_spec.rb

```diff
@@ -11,42 +12,53 @@
 # we effectively get all that testing for free! Here we'll
 # only test things that are different.
 
-xdescribe "OnlineOrder" do
+describe "OnlineOrder" do
   describe "#initialize" do
     it "Is a kind of Order" do
       # Check that an OnlineOrder is in fact a kind of Order
-
-      # Instatiate your OnlineOrder here
-      # online_order =
-      # online_order.must_be_kind_of Grocery::Order
+      #Instatiate your OnlineOrder here
+      online_order = Grocery::OnlineOrder.new(1, {}, 123)
+      online_order.must_be_kind_of Grocery::Order
     end
 
     it "Can access Customer object" do
       # TODO: Your test code here!
+      online_order = Grocery::OnlineOrder.new(1, {}, 234)
+      (online_order.customer_id).wont_be_nil
     end
 
     it "Can access the online order status" do
       # TODO: Your test code here!
+      online_order = Grocery::OnlineOrder.new(1, {}, 23)
+      (online_order.fulfillment_status).wont_be_nil
     end
   end
 
   describe "#total" do
     it "Adds a shipping fee" do
       # TODO: Your test code here!
+      online_order = Grocery::OnlineOrder.new(1, {"bulk spaghetti" => 10}, 23)
+      (online_order.total).must_equal 20.75
     end
 
     it "Doesn't add a shipping fee if there are no products" do
       # TODO: Your test code here!
+      online_order = Grocery::OnlineOrder.new(1, {}, 23)
+      (online_order.total).must_equal 0
     end
   end
 
   describe "#add_product" do
     it "Does not permit action for processing, shipped or completed statuses" do
       # TODO: Your test code here!
+      online_order = Grocery::OnlineOrder.new(1, {"pigs feet" => 3.90}, 23, "completed")
+      proc{online_order.add_product("jell-o", 0.99)}.must_raise ArgumentError
     end
 
     it "Permits action for pending and paid satuses" do
       # TODO: Your test code here!
+      online_order = Grocery::OnlineOrder.new(1, {"pigs feet" => 3.90}, 23, "pending")
+      online_order.add_product("jell-o", 0.99).must_equal true
```

Comment:
You should check the paid status here as well, since that's what the test's name advertises.